openapi-generator/src/test/resources/3_0/controller-enum.yml

```diff
@@ -103,6 +103,7 @@ paths:
           enum:
             - "year"
             - "rating"
+            - "big_rating"
```

Comment:
why is this change necessary in the PR to update versions?

Reply:
this case need to test openapi-generator 7.5.0. Problem ws with generating enums, which contains `_` symbol.

`if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_'))` - fix it